pkg/core/localize/locales/en/cmd/setting.en.toml

```diff
@@ -0,0 +1,88 @@
+[setting.cmd.description.short]
+one = 'Configure settings of a Service Registry instance'
+
+[setting.cmd.description.long]
+one = '''
+Configure settings of a Service Registry instance
```

Comment:
Maybe a longer description for this

Reply:
We have dedicated documentation team that works on descriptions for the commands. 
It is good to write something long but certainly not required.

Reply:
Hey folks, 
This is an admin only-feature in the Apicurio Registry API and RHOSR UI. Assuming this is admin-only also in RHOAS CLI? 

I can create a separate PR to update the help doc with more info (e.g. admin/instance owner feature, available settings, etc.).

Reply:
Good point. We might actually add some extra checks to see if user is admin (assuming that non admin would not get useful error.